ORM/ORMFundamentals/Repositories/IOrderRepository.cs

```diff
@@ -0,0 +1,21 @@
+﻿using ORMFundamentals.Entities;
+using System.Linq.Expressions;
+
+namespace ORMFundamentals.Repositories;
+
+public interface IOrderRepository
```

Comment:
To avoid boilerplate code for Add/Update/Delete strongly recommend to use generic _Repository_, you can refer to my example [here](https://github.com/dotnet-fizzyy/Diploma/blob/V2/WebAPI/WebAPI/Infrastructure/Postgres/Repository/BaseCrudRepository.cs#L13)